modules/prediction/predictor/extrapolation/extrapolation_predictor.cc

```diff
@@ -51,19 +52,41 @@ void ExtrapolationPredictor::Predict(
 
 void ExtrapolationPredictor::PostProcess(Trajectory* trajectory_ptr) {
   // TODO(kechxu) handle corner cases
-  auto lane_search_result = SearchExtrapolationLane(*trajectory_ptr);
+  constexpr int kNumTailPoint = 5;
+  ExtrapolationPredictor::LaneSearchResult
+  lane_search_result = SearchExtrapolationLane(*trajectory_ptr, kNumTailPoint);
   if (lane_search_result.found) {
     ExtrapolateByLane(lane_search_result.lane_id, trajectory_ptr);
   } else {
-    constexpr int kNumTailPoint = 5;
     ExtrapolateByFreeMove(kNumTailPoint, trajectory_ptr);
   }
 }
 
 ExtrapolationPredictor::LaneSearchResult
-ExtrapolationPredictor::SearchExtrapolationLane(const Trajectory& trajectory) {
+ExtrapolationPredictor::SearchExtrapolationLane(
+    const Trajectory& trajectory, const int num_tail_point) {
+  constexpr double radius = 1.0;
+  constexpr double angle_diff_threshold = M_PI / 3.0;
+  int num_trajectory_point = trajectory.trajectory_point_size();
+
   LaneSearchResult lane_search_result;
-  // TODO(kechxu) implement
+  for (int i = num_trajectory_point - 1;
+       i >= num_trajectory_point - num_trajectory_point; --i) {
```

Comment:
What do you mean by `num_trajectory_point - num_trajectory_point`, 0?

Reply:
fixed.